lgc/patch/Patch.cpp

```diff
@@ -138,6 +138,10 @@ void Patch::addPasses(PipelineState *pipelineState, lgc::PassManager &passMgr, T
   passMgr.addPass(PatchWorkarounds());
   passMgr.addPass(PatchCopyShader());
   passMgr.addPass(LowerVertexFetch());
+  // Enabling uber shader will generate two external functions, enable inline to optimize
+  if (pipelineState->getOptions().enableUberFetchShader)
+    passMgr.addPass(AlwaysInlinerPass());
+
```

Comment:
No. AlwaysInliner is already run below. Please avoid complicating the pass pipeline like this.